### PR TITLE
Added remarks about VPN to peach.md

### PR DESCRIPTION
**Pull request by lasermind**
As we discussed, adding a note about using Peach over a VPN service to enhance a users network privacy would be a good idea. Here is my take on this, I hope I found a reasonable place to add it to the existing text.

### CODE REVIEW

**Comment by lasermind** [comment]
An additional thought: what's true for Peach here, holds also true for visiting websites like HodlHodl. Maybe remarks about using a VPN service to improve a user's network privacy should also be made in other guides.

If you're interested I could have a look at this as well.

File: peach.md [hunk 1]
```diff
@@ -246,6 +246,8 @@ Many Bitcoiners prioritise Free and Open Source Software (FOSS) when selecting t
 ### Will Peach integrate Tor?
 At the moment, Tor integration is parked in their pipeline but not a priority.
 
+Because of that, a user is well advised to be on a [VPN](https://en.wikipedia.org/wiki/VPN_service) connection while interacting with the Peach app. Using a VPN is generally good practice on mobile phones and desktop devices. This way, the actual network IP of your device will not be visible to the services you connect to, thus gaining you network privacy, beneficial for all sorts of reasons. Fortunately, with providers like Mullvad VPN, for example, it's becoming more and more easy to use a good VPN even on mobile. It's not free though, it's a fiver a month – but it's worth it as soon as you learn to value your privacy, especially while making sensitive financial transactions. Big privacy bonus: you can even pay Mullvad via cash by mail (yes it works!) or in BTC.
```
Comment (bitlevi) [inline peach.md:249]: Thanks for creating this PR so fast! It is a very clever addition to the guide. 

Personally, I would keep it short and cut at "even on mobile" and link the episode of journey to sovereignty about VPN if the reader wants to learn more about the subject. 

What do you think? Feel free to push back 🙌

The best feedback will likely come from QnA 🤫

Comment (BitcoinQnA) [reply peach.md:249]: I agree here. Let's remember that the focus here is Peach and letting the know that VPNs are a best practice. The existing hyperlink can explain more on what a VPN is.

Only thing I would add to update the sentence `Using a VPN is generally good practice on mobile phones and desktop devices.` to `Using a VPN like [Mullvad](https://mullvad.net) is generally good practice on mobile phones and desktop devices.`

**Comment by BitcoinQnA** [comment]
> An additional thought: what's true for Peach here, holds also true for visiting websites like HodlHodl. Maybe remarks about using a VPN service to improve a user's network privacy should also be made in other guides.
> 
> If you're interested I could have a look at this as well.

I would love for you to explore this too. Thank you!

**Comment by BitcoinQnA** [comment]
Ready to merge after implementing feedback provided. Thank you @lasermind

**Comment by lasermind** [comment]
Thanks for the feedback, no problem, we can keep it short!

I also was torn: on one hand, I did not want to advertise Mullvad too much, on the other hand I did not want a reader to leave without learning about the benefits of Mullvad. This way now, I'm also very happy: short paragraph and Mullvad mentioned. 👍

Made the suggested changes to my `peach.md`; if it's fine for you, it's ready to be pulled.

**Comment by lasermind** [comment]
Alright, done that! 

If you want to edit further nuances you can: I initially activated "allow edits by maintainers" on purpose. We don't need to play Chinese whispers here day in and day out. 😉

**Comment by bitlevi** [comment]
Smashing, brilliant thanks! 🚀

**Comment by BitcoinQnA** [comment]
@lasermind thank you for the updates
@bitlevi appreciate the time taken to review

**Pull request merged**